src/AbstractFunctionReferenceFixer.php

```diff
@@ -67,7 +67,7 @@ protected function find($functionNameToSearch, Tokens $tokens, $start = 0, $end
         $matches = $tokens->findSequence($candidateSequence, $start, $end, false);
         if (null === $matches) {
             // not found, simply return without further attempts
-            return;
+            return null;
```

Comment:
> still, I wanted to solve issue raised by SCA.

The static analyzer is wrong IMO, since we both agree the semantics of what the function returns is the same, so by definition, it was already correct.